### ISSUE TITLE
Add `reportUnscopedDisables` flag

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#2292]
Ref: https://github.com/stylelint/stylelint/pull/2232#discussion_r96887239

Report disables not scoped to one or more rules.

Treated as a similar violation to `reportNeedlessDisables`.

This flag (`reportUnscopedDisables`) would trigger a violation for the following source:

```css
/* stylelint-disable */
```

As no rule(s)/plugin(s) is/are listed.

**Comment by kmiyashiro** [comment]
Hmm, the changelog says that `stylelint-disable-reason` is deprecated and to consider contributing to this instead, but it seems like they are two different rules. `stylelint-disable-reason` checks for a separate comment before or after the disable comment, while this checks for 0 rules in the disable comment.

**Comment by jeddy3** [comment]
@kmiyashiro Thanks for spotting that. Yes, you're right. `reportUnscopedDisables` is meant as an alternative to the `stylelint-disable-no-empty` proposal that got mothballed. I've posted [a PR](https://github.com/stylelint/stylelint/pull/2393) to correct the CHANGELOG.

**Comment by Mouvedia** [comment]
@jeddy3 is this ready to implement?

**Comment by Mouvedia** [comment]
@ybiquitous is this ready to implement?

**Comment by ybiquitous** [comment]
Hum, I want to confirm again if this new flag is really needed by people because this issue is quite old.

@jeddy3 Please let us hear your thoughts. 🙏🏼

**Comment by jeddy3** [comment]
I believe it's still valid as a [`report*`](https://stylelint.io/user-guide/configure#report) flag. Wordpress [wanted the feature not so long ago](https://github.com/Automattic/wp-calypso/pull/67477).

**Comment by ybiquitous** [comment]
Thanks for the confirmation. I've transitioned this issue to *ready to implement*.
